source/PlayerInfo.h

```diff
@@ -180,6 +180,17 @@ class PlayerInfo {
 	
 	// Access the "condition" flags for this player.
 	int64_t GetCondition(const std::string &name) const;
+	// Copy conditions matching the prefix into the given map.
+	void GetConditions(std::map<std::string, int64_t> &targetMap, const std::string &prefix) const;
+	// Retrieve a sum (addition) of all "condition" flags starting with the
+	// given prefix.
+	int64_t GetConditionSum(const std::string &prefix) const;
+	// Set a "condition" flag to the given value. Returns true on success,
+	// false on failure.
+	bool SetCondition(const std::string &name, int64_t value);
+	bool AddCondition(const std::string &name, int64_t value);
+	bool EraseCondition(const std::string &name);
```

Comment:
I'm guessing having a return type is setting up for later.

Reply:
Yes, that is correct, this is for when we get read-only conditions.